### PR TITLE
[native]Add system config for per-driver cpu time slicing enforcement

### PR DESCRIPTION
**Pull request by xiaoxmeng**
Add system config got per-driver cpu time slicing enforcement and set the
corresponding config in velox query config accordingly

### CODE REVIEW

File: presto-native-execution/presto_cpp/main/common/Configs.cpp [hunk 1]
```diff
@@ -573,6 +574,10 @@ bool SystemConfig::useLegacyArrayAgg() const {
   return optionalProperty<bool>(kUseLegacyArrayAgg).value();
 }
 
+uint32_t SystemConfig::driverCpuTimeSliceLimitMs() const {
+  return optionalProperty<bool>(kDriverCpuTimeSliceLimitMs).value();
```
Comment (tanjialiang) [inline presto-native-execution/presto_cpp/main/common/Configs.cpp:578]: return optionalProperty<uint32_t>(kDriverCpuTimeSliceLimitMs).value();

File: presto-native-execution/presto_cpp/main/common/Configs.cpp [hunk 2]
```diff
@@ -752,6 +760,8 @@ void BaseVeloxQueryConfig::updateLoadedValues(
       {QueryConfig::kMaxPartitionedOutputBufferSize,
        systemConfig->capacityPropertyAsBytesString(
            SystemConfig::kDriverMaxPagePartitioningBufferSize)},
+      {QueryConfig::kDriverCpuTimeSliceLimitMs,
+       std::to_string(systemConfig->driverCpuTimeSliceLimitMs())},
```
Comment (tanjialiang) [inline presto-native-execution/presto_cpp/main/common/Configs.cpp:764]: shall we use folly::to\<std::string\>() ?

Also I don't quite understand this method. Why are we putting these query config into system config while initialize the system config? Thanks

**Review by tanjialiang — COMMENTED** [review]
Thanks for the change.

File: presto-native-execution/presto_cpp/main/common/Configs.cpp (hunk 2 above)
Comment (xiaoxmeng) [reply presto-native-execution/presto_cpp/main/common/Configs.cpp:764]: Yeah, we probably doesn't need system config. Removed.

**Pull request closed without merging**